### DISCUSSION TITLE
Popping open our last check-in of the year  :champagne: [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#42302] [New to GitHub]
A toast to all of you who helped make our community a helpful, vibrant, and fun place to discuss all things GitHub in 2022 - we couldn’t have done it without you. This is my last check-in for the year so cozy up! 

🌟**Community Superstars**: A round of applause for these community citizens 👏
* @mohamedzhioua - For a great and robust [answer to the difference between SSH keys and deploy keys](https://github.com/orgs/community/discussions/40788)! And then continuing to help ♥
* @MitkoVtori & @DSDMark - Loved both of their responses for the [user asking how to create a profile ReadME](https://github.com/orgs/community/discussions/40518) - @mitkovtori gave the steps and @DSDMark followed up with a doc  📃 - two heads are better than one!
* @DianVK - [Wanted to move two repositories into another one](https://github.com/orgs/community/discussions/41444) and gave an in-depth view of their thought process so other members could help. Thank you! 

📚 **Rousing Reads**: GitHub blog goodies 🍭
* **Release Radar October 2022 Edition**: While this may be delayed, we wanted to take a moment to celebrate those that did release during Hacktoberfest 👻. [Read up](https://github.blog/2022-12-09-release-radar-october-2022/) on these projects, learn about their breaking changes, and view some fantastic statistics from the open source community from @mishmanners.
* **Hello from GitHub’s new Chief Product Officer**: We’re introducing our new Chief Product officer, @shainbal, get to know her in her [blog post](https://github.blog/2022-12-08-hello-from-githubs-new-chief-product-officer/). 👷
* **ICYMI: A look back at GitHub Universe 2022** - One [last Universe post](https://github.blog/2022-12-05-icymi-a-look-back-at-github-universe-2022/) from us to close out the year 🪐.  @lclindeman gets you caught up on everything we announced and shares what else happened during this year’s GitHub Universe conference that took place November 9-10.

✨ **Start the new year, a little less new to GitHub**: Brush up on your skills or learn something new before 2022 ends! 
* **How to choose (and contribute to) your first open source project**: The fun of open source is contributing to others, this [collection](https://github.com/collections/choosing-projects) has impactful projects for you to participate in 💃! 
* **How I Got Started with GitHub**: Read about how I learned GitHub and the resources I used in my [blog](https://github.com/orgs/community/discussions/32792). 
* **Learn how to use GitHub with our checklist**: No one knows better than us how much there is to learn in GitHub so we were inspired to create a new user checklist to help you get the hang of things and begin learning. New user or looking to refine the basics? [Begin working on our checklist](https://github.com/orgs/community/discussions/27367) ✅!

Cheers to a great year - may 2023 bring more solutions, projects, and releases our way. See you then 🎉.

**Comment by MitkoVtori** [comment]
I can't trust I am on the list again! 🤣
Love ya 💖

**Comment by Hassan0573** [comment]
😱😱
